modules/aws/jumpbox/aws_cleanup.sh.tmpl

```diff
@@ -2,6 +2,12 @@
 
 export REGION="${region}"
 export VPC_ID="${vpc_id}"
+export REPO_PREFIX="${registry_name}"
+
+echo 'Destroying ECR repositories...'
+for repo in $(aws ecr describe-repositories --region $REGION --query 'repositories[*].repositoryName' --no-paginate --output text); do
+    [[ "$repo" =~ ^$REPO_PREFIX/.* ]] && aws ecr delete-repository --repository-name $repo --region $REGION --force
+done
```

Comment:
in AWS, the `tctl sync` will create one repository for each image, but that's not managed by terraform and was never cleaned up.
This code makes sure all repositories that could have been created are properly removed.